apps/user/endpoints/v1/auth.py

```diff
@@ -37,31 +44,46 @@ def auth_post(body: auth.Auth, response: Response):
             'client_secret': os.environ['CLIENT_SECRET'],
         },
     )
-
+    if ms_response.status_code != status.HTTP_200_OK:
+        return JSONResponse(
```

Comment:
Is there a special reason you use JsonResponse and not just raise exceptions?

Reply:
Raising HTTPException gives pydantic.error_wrappers.ValidationError here because `response_model=auth.AccessToken` enforces the validation of the response.

I found https://fastapi.tiangolo.com/advanced/additional-responses/, and I decided using it.

Reply:
how about using `raise HTTPException`? I remember that raising exception causes fastapi to create a separate model when an error occurs.

Reply:
Specifying `respones={500: {'model': Message}` makes responses example in the swagger document. And returning JSONResponse fits the model. Also, returning JSONResponse doesn't make redundant error logs on the server which can be seen when raising an error.